website/docs/enterprise/getting-started/install-enterprise-azure.mdx

```diff
@@ -0,0 +1,276 @@
+---
+title: Azure and Weave GitOps Enterprise Installation
+hide_title: true
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Azure and Weave GitOps Enterprise Installation<TierLabel tiers="Enterprise" />
+
+Once you successfully create your Kuberbetes cluster in Azure Marketplace, follow these steps to Install Weave GitOps Enterprise. 
+
+## 1. Choose the “GitOps” option in the Marketplace
+
+Search for Weave GitOps Enterprise in the "Extensions + Applications" of the [Azure Marketplace](https://portal.azure.com/signin/index/). Click the "GitOps" option. This will take you to a screen that presents a first-class item called `Type: Flux v2`.
+
+Click GitOps => Create.
+
+Add the config name, namespace (default), scope: cluster, type (Flux v2), and continuous reconciliation option. Your entries should look like this:
+- Configuration: flux-system
+- Namespace: flux-system
+- Scope: Cluster
+
+All of the displayed properties for the Flux objects screen are the same as what you'd supply to Flux bootstrap.
+
+Click "Next", then add:
+- Source Kind: Git repository
+- Repository URL: [your repository URL here]
+- Reference Type: Branch
+- Repository Type: Private
```

Comment:
This is still a useful guide but WGE _should_ auto-install flux now when you try and install it.

Reply:
@foot Should we have the auto-install as the main/default doc text but reference these lines as "if for some reason WGE doesn't install Flux automatically ..."—like a fallback option?

Reply:
WGE won't install if the flux auto-installation fails, so could phrase it like that. Good backup option to work around some error there.

The guide to creating a source / kustomization is great though, the non-azure instructions ask you to `flux bootstrap` which does this initial git/ks creation for you. It should be done at some point before you use many WGE features like rendering templates out to git, which assumes that git repo will be being sync'd back into the cluster.

Reply:
What do you think about maybe moving it down the bottom to a troubleshooting section? e.g. The happy install path shouldn't require the user do this manually.

---

#### Troubleshooting

WGE will try and automatically install flux on a new cluster, if this fails for some reason or if you need a more custom flux installation you can manually install it before installing WGE.

Reply:
Good idea, I did that